server.js

```diff
@@ -1,8 +1,23 @@
 const express = require('express')
 const app = express()
+const mongoose = require('mongoose')
+const URI = 'mongodb+srv://dhruvk:q6GkQB4Kg3jvSHFd@dhruv-bike-info.dga5pzc.mongodb.net/?retryWrites=true&w=majority'
+
+mongoose.connect(URI)
+.then(()=>{
+    app.get('/',(req,res)=>{
```

Comment:
Consider adding a separate endpoint to check the database connection status instead of overloading the root endpoint. [medium]